### PR TITLE
Database/add.implement.db

### PR DESCRIPTION
**Pull request by TheYisusGithub**
This is the implementation the db in mysql server and read de configuration the setting.

### CODE REVIEW

File: src/main/java/com/fundation/webservice/database/Query.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+            e.getMessage();
+            System.out.printf(String.valueOf(e));
+        }
+    }
+
+    /**
+     * This method show informations from the table criteria.
+     */
+    public void getAllData() {
+        //List<String> infsavedb = new ArrayList<String>();
```
Comment (JosueRodriguezGarcia) [inline src/main/java/com/fundation/webservice/database/Query.java]: Delete annotated code

File: src/main/java/com/fundation/webservice/common/Util.java [hunk 2]
```diff
@@ -63,6 +77,10 @@ private static void initUtil() {
         System.out.println("existo");
```
Comment (JosueRodriguezGarcia) [inline src/main/java/com/fundation/webservice/common/Util.java]: delete system.out.println()

File: src/main/java/com/fundation/webservice/database/DBConnection.java [hunk 3]
```diff
@@ -30,34 +33,60 @@ private DBConnection() {
...
+            }
+            catch (Exception ex) {
+                ex.printStackTrace();
+            }
+            System.out.println(WEBSERVER_DB);
             conn = DriverManager.getConnection("jdbc:mysql://"+HOST_NAME+":"+PORT_CONNECTION+"/"+WEBSERVER_DB, USER_NAME, USER_PASSWORD);
-            System.out.println("existo");
+            System.out.println("---Conneted---");
         } catch (SQLException e) {
+            System.out.println("--Fail---");
```
Comment (JosueRodriguezGarcia) [inline src/main/java/com/fundation/webservice/database/DBConnection.java]: delete system.out.println()

File: src/main/java/com/fundation/webservice/database/DBConnection.java [hunk 4]
```diff
@@ -30,34 +33,60 @@ private DBConnection() {
...
+            System.out.println(e);
```
Comment (JosueRodriguezGarcia) [inline src/main/java/com/fundation/webservice/database/DBConnection.java]: delete system.out.println()

File: src/main/java/com/fundation/webservice/database/Query.java [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+/**
+ * Implements insert in a table .
+ *
+ * @author Jesus Menacho
+ * @version 1.0
+ */
+public class Query {
+    /**
+     * This method inster informations from name and json to Data Base
```
Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/database/Query.java:28]: please redact better the comment.
Check grammar and orthography.

File: src/main/java/com/fundation/webservice/database/Query.java [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+        String sql = "INSERT INTO savedb(checksum,timeDate) VALUES(?,?)";
+        try {
+            Connection connection = DBConnection.getInstance().getConnection();
+            PreparedStatement statement = connection.prepareStatement(sql);
+            statement.setString(1, checksum);
+           statement.setString(2, date);
+            statement.executeUpdate();
+        } catch (SQLException e) {
+            e.getMessage();
+            System.out.printf(String.valueOf(e));
```
Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/database/Query.java:40]: delete Sout

File: src/main/java/com/fundation/webservice/database/Query.java [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+     */
+    public void getAllData() {
+        //List<String> infsavedb = new ArrayList<String>();
+        String sql = "SELECT * FROM savedb";
+        try {
+            Connection conn = DBConnection.getInstance().getConnection();
+            Statement statement = conn.createStatement();
+            ResultSet result = statement.executeQuery(sql);
+            while (result.next()) {
+
```
Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/database/Query.java:55]: delete space please.

File: src/main/java/com/fundation/webservice/database/Query.java [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+                System.out.println(result.getInt("id") + "\t" + result.getString("checksum") + "\t" + result.getDate("timeDate") );
```
Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/database/Query.java:56]: delete Sout

File: src/main/java/com/fundation/webservice/database/Query.java [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+        try {
+            Connection connection = DBConnection.getInstance().getConnection();
+            PreparedStatement statement = connection.prepareStatement(sql);
+            statement.setString(1, ID);
+            statement.executeUpdate();
+        } catch (SQLException e) {
+            e.getMessage();
+        }
+    }
+
```
Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/database/Query.java]: add comment please

**Pull request merged**